core/src/test/java/tech/tablesaw/io/csv/CsvWriterTest.java

```diff
@@ -54,7 +58,135 @@ public void dateFormatter() throws IOException {
   }
 
   @Test
-  public void dateTimeFormatter() throws IOException {
+  void printFormatter1() throws IOException {
```

Comment:
could we give these more descriptive names like `printFormatter_number`, `printFormatter_boolean`, etc.